source/guides/network/maintaining-a-symbol-node.rst

```diff
@@ -0,0 +1,125 @@
+.. post:: 06 Jun, 2021
+    :category: Network
+    :excerpt: 1
+    :nocomments:
+
+#########################
+Maintaining a Symbol Node
+#########################
+
+Once you have created a |codename| node, be it :doc:`manually <running-a-symbol-node-manually>` or :doc:`using Symbol bootstrap <running-a-symbol-node>`, the node is **mostly autonomous**. But there is still a little bit of maintenance required, explained in this guide.
+
+- Regular nodes only need to take care of the :ref:`maintain-server-version` section.
+
+- :ref:`Voting nodes <voting-node-program>` also need to read the :ref:`maintain-voting-keys` section.
+
+.. _maintain-server-version:
+
+Server updates
+**************
+
+New |codename| server versions will be released **periodically**. If you want to benefit from the added features and bugfixes you will need to **update your node** to use the latest server version.
+
+Additionally, if you have enrolled in any of the :doc:`../../concepts/reward-programs` keeping up with server versions is a **mandatory requirement**.
+
+The main version to keep track of is the **Catapult Server version**, since this component is used by all :ref:`peer nodes <peer-node>`. Additionally, :ref:`API nodes <api-node>` need to keep track of the **REST component version** too.
+
+- **Finding out what are the latest versions**:
+
+  Whenever there is a new server version released it will be announced through the NEM |forum| and the |slack| channel. You can also keep an eye on the different |github| repositories and :doc:`the compatibility matrix page <../../compatibility>` to know the latest Server and REST versions.
+
+- **Finding out your node versions**:
+
+  You can find the versions used by your node in the `Symbol Explorer nodes list <http://explorer.symbolblockchain.io/nodes>`__.
+
+  Additionally, if your node is an :ref:`API node <api-node>`, you can also query it directly:
+
+  - **Server version**: ``/node/info`` endpoint, ``version`` property. `See an example <http://ngl-dual-104.symbolblockchain.io:3000/node/info>`__.
```

Comment:
I think this should be reading from `config-node.properties` from the [localnode].version. Peer only node won't have rest

Reply:
That's what the sentence above means: `Additionally, if your node is an API node you can also query it directly.`
According to @gimer finding the server's version was not as easy as looking at a config file :(

Reply:
well. rest api is reading from this config property anyway

Reply:
Rest is not reading the version from the file, it gets the /node/info from the server API call.  Only the node public key for the delegate harvesting setup is loaded from the file